tests/pykafka/test_balancedconsumer.py

```diff
@@ -264,8 +264,8 @@ def test_no_partitions(self):
         consumer._decide_partitions = lambda p: set()
         consumer.start()
         self.assertFalse(consumer._running)
-        with self.assertRaises(NoPartitionsForConsumerException):
-            consumer.consume()
+        res = consumer.consume()
+        self.assertEqual(res, None)
```

Comment:
This test seems to have a contradiction now: in L266 we demand that the consumer has in fact stopped, but in that case L268 should see an exception?